src/ReTestItems.jl

```diff
@@ -16,6 +16,7 @@ export TestSetup, TestItem, TestItemResult
 const RETESTITEMS_TEMP_FOLDER = mkpath(joinpath(tempdir(), "ReTestItemsTempLogsDirectory"))
 const DEFAULT_TESTITEM_TIMEOUT = 30*60
 const DEFAULT_RETRIES = 0
+const DEFAULT_MEMORY_THRESHOLD = 0.99
```

Comment:
Should we perhaps be more conservative here and set this to `0.9` or `0.95`? It seems like we might still run into issues if, for example, memory was 98% and then we kick off another test item to run.

Reply:
idk... i was wondering if this should even just not be on by default (i.e. set to 1)

in raicode we should be setting it ourselves anyway

Reply:
Yeah, that's fair. I'd be fine "leaving it off" as `1.0`.

Reply:
i think this is probably useful enough and 99% conservative enough that i'll leave this as the default for now -- we could always change it later